javalin/src/main/java/io/javalin/core/validation/Rules.kt

```diff
@@ -0,0 +1,4 @@
+package io.javalin.core.validation
+
+data class NullableRule<T>(val fieldName: String, val check: (T?) -> Boolean, val invalidMessage: String)
+data class Rule<T>(val fieldName: String, val check: (T) -> Boolean, val invalidMessage: String)
```

Comment:
You need to add an upper bound of `Any` to T as the default upper bound is `Any?`. Rule could still accept null otherwise.

Reply:
I needed that if I wanted to convert a `NullableValidator` into `Validator`, but not when going the other way (which is what I ended up doing).